internal/photoprism/import_worker.go

```diff
@@ -152,7 +152,7 @@ func ImportWorker(jobs <-chan ImportJob) {
 				continue
 			}
 
-			done := make(map[string]bool)
+			done := make(map[string]*struct{})
```

Comment:
`bool` size is max. 1 byte (can be optimized to 1 bit), while `*struct{}` is a pointer and would take 8 bytes on a 64bit system. What you probably intended is `map[string]struct{}`, which indeed has a size of 0.

Reply:
Yup you're right. Intention was to use the type that takes no memory.